src/09_dictionaries.py

```diff
@@ -36,12 +36,31 @@
 # Add a new waypoint to the list
 # YOUR CODE HERE
 
+newEntry = {
+    "lat":100,
+    "lon":80,
+    "name":"yet another place"
+}
+
+waypoints.append(newEntry)
+
+# print(waypoints)
+
 # Modify the dictionary with name "a place" such that its longitude
 # value is -130 and change its name to "not a real place"
 # Note: It's okay to access the dictionary using bracket notation on the
 # waypoints list.
 
 # YOUR CODE HERE
 
```

Comment:
no change for lon here. line 50 is asking for you to update longitude and the name. please update this and push the changes.